db/schema.rb

```diff
@@ -10,10 +10,22 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_05_18_193518) do
+ActiveRecord::Schema[7.0].define(version: 2023_05_26_230141) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "actors", force: :cascade do |t|
+    t.string "name"
+    t.integer "age"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "actors_movies", id: false, force: :cascade do |t|
```

Comment:
Since you called the table `actors_movies`, you're going to need to make sure "actors" is plural anywhere you reference (which is why I think you were having trouble adding actors to your movies in your spec. Rails wasn't recognizing the relationship because it wasn't defined quite correctly in the model. I left on comment on your models to show how it should've looked.